### ISSUE TITLE
Add all helm chart templates in onboarding process

### ISSUE THREAD

**Issue by checkelmann** [keptn/keptn#558]
Hi,

when we are onboarding a project and importing the helm chart, we have only the ability to add the values, deployment and service template to keptn, and need to add all other helm templates afterward manually to the created branches in git.

As our deployments consist of more template files and template helper functions (like ingress definitions, persistent volume claims, secrets etc...) it would be good if you can just define a helm chart directory and keptn will onboard all files nested under the template folder like

```
keptn onboard service --project=sockshop --chart=path/to/chart
```

Thank you!

**Comment by johannes-b** [comment]
Hi Christian, 
thanks for the feature request! 
We are aware of the limitation abd have a cli extension (as shown above) in mind. After changing the underlying keptn architecture for storing files in git and other git-based implementations, we will address this concern.

**Comment by johannes-b** [comment]
Hi Christian, 

we implemented your feature request and it is already available in 0.5.0.beta. I recommend to wait for the official 0.5.0 release, which is around the corner.

For onboarding a service, you can now use the Keptn CLI with the `--chart` flag, that allows you to specify your Helm chart:
```
keptn onboard service <your-service> --project=<your-project> --chart=<path/to/your-service.tgz>
```
This onboards the service based on the provided chart.

(Based on the feature support in Keptn 0.5.0.beta, I will close the issue.)
